opm/autodiff/VFPProdProperties.cpp

```diff
@@ -110,4 +110,143 @@ bool VFPProdProperties::hasTable(const int table_id) const {
 }
 
 
+std::vector<double>
```

Comment:
What is the main difference of these new function with detail::bhp and detail::thp?

Reply:
this is based on fixed wfr, gfr, alq and thp, providing a group of flo rate values, calculate a group of bhp values.  It is just more efficient, since you do not need to recalculate the wfr and gfr for each group of rates, then do the table search.